client/layout/logged-out.jsx

```diff
@@ -32,6 +33,7 @@ module.exports = React.createClass( {
 				<Masterbar />
 				<div id="content" className="wp-content">
 					<NoticesList id="notices" notices={ notices.list } />
+					<GlobalNotices id="notices" />
```

Comment:
Good point!
`GlobalNotices` is very rudimentary for now, it copies most of NoticesList, but needs a better design.